### ISSUE TITLE
Typo in twisted/conch/openssh_compat/factory.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5580]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5580|
|Type|defect|
|Created|2012-03-28 15:24:36Z|

The module docstring says "modile", means "moduli".

Discovered by teratorn, but he didn't feel like filing a ticket.

Attachments:

* [fix.modile.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/fee/fee055b039d2e9ce3396eeecb5d34a5659678326/57e3bbb66d55ddc0717d703f8f607ddf220e83c1.patch) (402 bytes) - added by zomux on 2012-03-31 15:08:36Z - 
* [ticket5580.4.3.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/fee/fee055b039d2e9ce3396eeecb5d34a5659678326/7fb8881eee873c539e4bc5abb01147d1bb66cae8.patch) (402 bytes) - added by zomux on 2012-04-03 01:05:58Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5580 5580
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__easy easy
time__1332948276000000 1332948276000000
changetime__1333819551000000 1333819551000000
version__None None
owner__ 
cc__z3p cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="zomux's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|zomux commented|
|-|-|

just modified the word modile :D

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to zomux|
|-|-|

Thanks for your contribution. Your patch calls it `module` but this shouldn't this be `moduli`?

**Comment by twisted-trac** [comment]
|<img alt="zomux's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|zomux commented|
|-|-|

Replying to [thijs](#note_3):
> Thanks for your contribution. Your patch calls it `module` but this shouldn't this be `moduli`?

Sorry , should it be 'moduli'?    It was my first contribution

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

> Sorry , should it be 'moduli'? It was my first contribution

Yea.  It's the faux-Latin plural of "modulus".

**Comment by twisted-trac** [comment]
|<img alt="zomux's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|zomux removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra commented|
|-|-|

This was applied in r34125 but not closed by trac.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34125]) Apply ticket5580.4.3.patch: Correct a typo in the docstring of t.conch.openssh_compat.factory.

Author: zomux
Reviewer: exarkun, thijs
Fixes: [#5580](https://github.com/twisted/twisted/issues/5580)
